Ginger/Ginger/Actions/ActionEditPages/ActScriptEditPage.xaml.cs

```diff
@@ -41,13 +41,14 @@ public partial class ActScriptEditPage : Page
         public ActScriptEditPage(GingerCore.Actions.ActScript Act)
         {
             InitializeComponent();
-            this.f = Act;
+            this.f = Act;            
```

Comment:
Can we use Act directly instead of coping it to variable. if its must, please give proper name to it

Reply:
needed replce with proper name